docs/aws/README.md

```diff
@@ -113,13 +143,22 @@ spec:
               cpu: 1
 EOF
 kubectl scale deployment inflate --replicas 5
-kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l control-plane=karpenter -ojson | jq -r ".items[0].metadata.name")
+kubectl logs -f -n karpenter $(kubectl get pods -n karpenter -l karpenter=controller -o name)
 ```
 
 ### Cleanup
 ```bash
 helm delete karpenter -n karpenter
 aws cloudformation delete-stack --stack-name Karpenter-${CLUSTER_NAME}
-aws ec2 describe-launch-templates | jq -r ".LaunchTemplates[].LaunchTemplateName" | grep Karpenter | xargs -I{} aws ec2 delete-launch-template --launch-template-name {}
+aws ec2 describe-launch-templates \
+    | jq -r ".LaunchTemplates[].LaunchTemplateName" \
+    | grep -i karpenter \
```

Comment:
I just realized that this is something that we should change, not necessarily for this PR. We delete any launch templates that have `karpenter` in them, meaning it's possible that a customer could install Karpenter in two clusters on the same account.  IIUC, this command would delete all Karpenter launch templates, which could have adverse affects on the another Karpenter installation on another cluster.

Would you be able to change this line to be the following? With this, I believe only the launch templates created by this installation would be deleted.
```
aws ec2 describe-launch-templates \
    | jq -r ".LaunchTemplates[].LaunchTemplateName" \
    | grep "Karpenter-${CLUSTER_NAME}" \
    | xargs -I{} aws ec2 delete-launch-template --launch-template-name {}
```

Reply:
Actually, nevermind me here. We create launch templates at allocation time, so this should be fine.

Reply:
I actually was worried about clobbering anything with karpenter in the name too. I originally had a more complex command that would look for the nodegroup stack launch template and delete it by ID. Really we only need to delete the karpenter-${CLUSTER_NAME} launch template we created and the rest will be deleted via `eksctl delete cluster` (if needed)